### PR TITLE
Updated hvd images + scripts

### PR DESCRIPTION
**Pull request by vfdev-5**
Related to #1305 

Description:
- added scripts to auto build and push images

Github actions yaml in another PR. 

Check list:
* [ ] New tests are added (if a new feature is added)
* [ ] New doc strings: description and/or example code are in RST format
* [ ] Documentation is updated (if required)

### CODE REVIEW

**Comment by vfdev-5** [comment]
@fco-dev please review

File: docker/push_all.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+image_name="msdp-apex-base"
+image_tag=`docker run --rm -it pytorchignite/${image_name}:latest -c 'python -c "import torch; import ignite; print(torch.__version__ + \"-\" + ignite.__version__, end=\"\")"'`
+
+for image_name in "msdp-apex-base" "msdp-apex-vision" "msdp-apex-nlp"
+do
+
+    docker push pytorchignite/${image_name}:latest
+    docker push pytorchignite/${image_name}:${image_tag}
+
+done
```
Comment (fco-dv) [inline docker/push_all.sh:55]: Once pushed, the local repository would maybe need a `clean` step. It also seems that multi-stage builds generate, `<none>` images:
```bash
docker images | grep 'pytorchignite\|<none>' | awk '{print $3}' | xargs docker rmi -f
```
what do you think ?

File: docker/msdp/build_all.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+curr_dir=$PWD
+cd $curr_dir/msdp
+
+set -xeu
+
+image_name="msdp-apex-base"
+
+docker build -t pytorchignite/${image_name}:latest -f Dockerfile.${image_name} .
+image_tag=`docker run --rm -it pytorchignite/${image_name}:latest -c 'python -c "import torch; import ignite; print(torch.__version__ + \"-\" + ignite.__version__, end=\"\")"'`
+docker tag pytorchignite/${image_name}:latest pytorchignite/${image_name}:${image_tag}
```
Comment (fco-dv) [inline docker/msdp/build_all.sh:21]: Maybe put `image_tag` in a for loop like for `main` and `hvd` scripts.

**Review by fco-dv — COMMENTED** [review]
@vfdev-5 great job! is there a need to separate the build/tag and push scripts ?

File: docker/push_all.sh (hunk 1 above)
Comment (vfdev-5) [reply docker/push_all.sh:55]: Yes, this is important for a local system, on the CI it is less important I think.

File: docker/msdp/build_all.sh (hunk 2 above)
Comment (vfdev-5) [reply docker/msdp/build_all.sh:21]: Yes, it was the first script I wrote. As there are 3 cases I though loop will not refactor a lot... 
OK, I can change that.

**Review by vfdev-5 — COMMENTED** [review]
Well, the reason for splitting build and push is that we may potentially would like perform locally only the build step.

Thanks for the review @fco-dv

**Pull request merged**